avocado/utils/pci.py

```diff
@@ -59,21 +59,23 @@ def get_pci_addresses():
         return addresses
 
 
-def get_num_devices_in_domain(domain):
+def get_num_interfaces_in_pci(dom_pci_address):
```

Comment:
Well there are human-interface-devices, there are infrared-interfaces, there are ata interfaces and one of the interfaces is network interface, so I don't associate interface necessarily with network.

On the other hand for example my integrated graphic card `00:02.0`, which is a single device, provides numerous interfaces like framebuffer, i2c, backlight, control, ....

I'm not sure whether `interface` is the best word, but I believe `device` is definitely incorrect. That's why I sent those changes in separate to get some responses on this...